token/services/db/dbtest/tokens.go

```diff
@@ -51,7 +51,7 @@ func TokensTest(t *testing.T, cfgProvider cfgProvider) {
 
 var tokensCases = []struct {
 	Name string
-	Fn   func(*testing.T, TestTokenDB)
+	Fn   func(context.Context, *testing.T, TestTokenDB)
```

Comment:
all these methods in the test don't need a context. as we said, when they call a method that requires a context (a method that is in an outside package), then we pass the background context.